spec/legacy/unit/journal_spec.rb

```diff
@@ -37,21 +37,19 @@
     end
   end
 
-  it 'should create should send email notification' do
-    ActionMailer::Base.deliveries.clear
+  it 'create should send email notification' do
     issue = WorkPackage.find(:first)
     if issue.journals.empty?
       issue.add_journal(User.current, 'This journal represents the creationa of journal version 1')
       issue.save
     end
-    user = User.find(:first)
-    assert_equal 0, ActionMailer::Base.deliveries.size
```

Comment:
Narf!

Of course that makes a difference! It is a shame that the specs for the notification mailer did not unveil it. Thanks for pointing out.

The difference here is that I will now send a notification based on the **update** setting in the **create** case. I should not be doing that.